src/containers/guest-home-page/cards-with-button/CardsWithButton.tsx

```diff
@@ -50,7 +51,11 @@ const CardsWithButton: FC<CardsWithButtonProps> = ({
             <Box className='dots' component='img' src={dots} />
           </Box>
           <TitleWithDescription
-            description={t(item.description)}
+            description={
+              <Typography variant='body2'>
+                {t(item.description as string)}
```

Comment:
Here we can receive undefined, because we have a type like this 
```
export interface AccordionItem {
  title: string
  description?: string
  content?: ReactNode
}
```
Maybe better to add some conditional replacement?
```suggestion
                {t(item.description || "")}
```